### PR TITLE
Show total awesomes on user page like old frontend

### LINKED ISSUE #108: Awesome count on user profiles
The old frontend displays the total awesome count on a user's profile:
image

This should be added to this frontend as well.

Requires changes to the API, and I suggest we create a new path for it, since it requires counting over a bunch of data and is only displayed in one place. Something like apiv2beta/users/$USER/awesome_count

### PR DESCRIPTION
**Pull request by rep-movsd**
Added UI in Header to show user total awesome count and total dweets on hover

Closes #108

### CODE REVIEW

**Review by lionleaf — APPROVED** [review]
This looks great! But I'll hold off on merging until the API has landed

File: src/Header.tsx
Comment (rep-movsd) [reply src/Header.tsx]: Removed debug code and changed endpoint as per suggestion on https://github.com/lionleaf/dwitter/pull/533

**Review by lionleaf — APPROVED** [review]
Looks good

**Pull request merged**
